backend/src/apiserver/config/config.json

```diff
@@ -15,7 +15,7 @@
   "ARCHIVE_CONFIG_LOG_FILE_NAME": "main.log",
   "ARCHIVE_CONFIG_LOG_PATH_PREFIX": "/artifacts",
   "InitConnectionTimeout": "6m",
-  "DefaultPipelineRunnerServiceAccount": "pipeline-runner",
+  "DEFAULTPIPELINERUNNERSERVICEACCOUNT": "pipeline-runner",
```

Comment:
nit: use either camel case or upper case with underscores, otherwise this is pretty hard to read.
(ideally we should be consistent on the naming, not sure which one is more conventional though)

Reply:
I agree that we should use `upper case with underscore`. But again, it is a breaking change that I don't want to introduce right now. Community has been testing Kubeflow v1.5 using KFP 1.8.0, if we are going to make this breaking change, we should aim for next release.

Reply:
Okay, that's reasonable.